integration-cli/docker_cli_secret_ls_test.go

```diff
@@ -0,0 +1,143 @@
+// +build !windows
+
+package main
+
+import (
+	"strings"
+
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/integration-cli/checker"
+	"github.com/go-check/check"
+)
+
+func (s *DockerSwarmSuite) TestSecretList(c *check.C) {
```

Comment:
This test could be merged with the other one (that would let use create and start one daemon, create once the secret and then list them with or without filters).